utils/libretime-backup.sh

```diff
@@ -1,9 +1,21 @@
 #!/bin/bash
 
-sound_folder='/srv/airtime/stor/'
-backup_folder='/home/example/backup/'
-psql_user='airtime'
-psql_password='airtime'
+if [ -z "$1" ]
+        then
+                ## Use config
+                backup_folder=~/libretime_backup/
+        else
+                ## User arg as config
+                backup_folder=$1
+fi
+
+
+airtime_conf_path=/etc/airtime/airtime.conf
+sound_folder=/srv/airtime/stor/
+
+psdl_db=$(grep dbname ${airtime_conf_path} | awk '{print $3;}' )
```

Comment:
You could also do `$(awk '/dbname/ {print $3}')` instead of piping grep to awk. awk is very much regex based under the hood anyway. This works just as fine though.

Reply:
Can you give me a complete example ? I don't understand 🤔

Reply:
```bash
psdl_db=$(awk '/dbname/ {print $3}' ${airtime_conf_path})
```